actions/sbom/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate SBOM for container images
+description: A GitHub Action for generating SBOM in spdx-json format
+author: Stefan Prodan
+branding:
+  color: blue
+  icon: command
+inputs:
+  version:
+    description: "Syft version"
+    required: false
```

Comment:
Isn't this required input?

Reply:
It is not, we get the latest version by default.

Reply:
Pinning to a version it is better done in each workflow I think.

Reply:
Ack, when I tried the snippet from the CLI PR, I got the impression a function argument was required. All good then :+1: